Add Homebrew Cask update to release process (#1671)
radicle-dev/radicle-upstream: DEVELOPMENT.md

by running any command that does a request to GitHub, like so: `hub api`.
You'll be asked to provide your GitHub login and the access token.

#### Homebrew CLI

> To update a formula at [Homebrew package manager](br), you'll need a

Reviewer: ```suggestion
To update a formula at [Homebrew package manager][br], you'll need a
```

Author: Thank you, both addressed in 46c8f48

> working `brew` CLI tool. The tool can use your GitHub _Persona access token_

Reviewer: ```suggestion
working `brew` CLI tool. The tool can use your GitHub _Personal access token_
```

> To update a formula at [Homebrew package manager][br], you'll need a
> working `brew` CLI tool. The tool can use your GitHub _Personal access token_
> from the previous step with
> `export HOMEBREW_GITHUB_API_TOKEN='<github-api-token>'` for authentification,
> in order to [setup a Cask repository fork, commit and push][bs] on your behalf.

Reviewer 2: ```suggestion
To update a formula at the [Homebrew package manager][br], you'll need a working 
`brew` CLI tool. 

The `brew`CLI requires a [GitHub Personal access token[gt] to [set up a Cask repository 
fork, commit and push][bs] on your behalf. You can make it available to brew with 
`export HOMEBREW_GITHUB_API_TOKEN='<github-api-token>'`

Note: this Github access token _must_ grant the `public_repo` scope.
```

Reviewer 2: @bzz let me know if you are happy with the suggestion I left or if you want to change it to your liking. Just don't want to assume your :+1: means green lights and step on your toes.
